index.bs

```diff
@@ -2785,27 +2817,28 @@ Note: The {{AttestationConveyancePreference}} enumeration is deliberately not re
 
 <div dfn-type="enum-value" dfn-for="AttestationConveyancePreference">
     :   <dfn>none</dfn>
-    ::  This value indicates that the [=[RP]=] is not interested in [=authenticator=] [=attestation=]. For example, in order to
+    ::  The [=[RP]=] is not interested in [=authenticator=] [=attestation=]. For example, in order to
         potentially avoid having to obtain [=user consent=] to relay identifying information to the [=[RP]=], or to save a
         roundtrip to an [=Attestation CA=] or [=Anonymization CA=].
+        If the [=authenticator=] generates an [=attestation statement=] that is not a [=self attestation=],
+        the [=client=] will replace it with a [=None=] attestation statement.
 
-        This is the default value.
+        This is the default, and unknown values fall back to the behavior of this value.
 
     :   <dfn>indirect</dfn>
-    ::  This value indicates that the [=[RP]=] prefers an [=attestation=] conveyance yielding verifiable [=attestation
-        statements=], but allows the client to decide how to obtain such [=attestation statements=].  The client MAY replace the
-        authenticator-generated [=attestation statements=] with [=attestation statements=] generated by an [=Anonymization CA=],
+    ::  The [=[RP]=] wants to receive a verifiable [=attestation statement=],
+        but allows the client to decide how to obtain such an [=attestation statement=].
+        The client MAY replace an authenticator-generated [=attestation statement=] with one generated by an [=Anonymization CA=],
         in order to protect the user's privacy, or to assist [=[RPS]=] with attestation verification in a heterogeneous ecosystem.
 
         Note: There is no guarantee that the [=[RP]=] will obtain a verifiable [=attestation statement=] in this case.
         For example, in the case that the authenticator employs [=self attestation=].
```

Comment:
This whole thing is a MAY, so there's not really any concrete expectation of what the client should do, but I'll add a clarification that this could happen if, for example, the client does nothing.

Reply:
Resolved now with latest changes.